.github/workflows/release.yml

```diff
@@ -0,0 +1,54 @@
+name: release
+
+on: release
+
+jobs:
+  release:
+    name: build and publish release assets
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+
+      - name: build release
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release --all-features
+
+      - name: package release assets
+        run: |
+          mkdir _dist
+          cp README.md LICENSE target/release/krustlet-wasi target/release/krustlet-wascc _dist/
+          cd _dist
+          tar czf krustlet-${{ steps.get_version.outputs.VERSION }}-linux-amd64.tar.gz README.md LICENSE krustlet-wasi krustlet-wascc
+
+      - name: generate checksums
+        run: |
+          cd _dist
+          sha256sum *.tar.gz > checksums.txt
+
+      - name: publish to crates.io
+        uses: actions-rs/cargo@v1
+        with:
+          command: publish
+          args: --token ${{ secrets.CargoToken }}
+
+      - name: upload tarballs to WABS
```

Comment:
nit: I assume `WABS` is Windows Azure Blob Storage? 
(I don't think that's the name anymore, and could we avoid the acronym?)

Reply:
absolutely. I just like saying the acronym wabs :)